sem_covid/services/pwdb_base_experiment.py

```diff
@@ -75,9 +77,8 @@ def data_validation(self, *args, **kwargs):
         pass
 
     def data_preparation(self, *args, **kwargs):
-        pwdb_json_dataset = json.loads(self.minio_adapter.get_object(config.PWDB_DATASET_LOCAL_FILENAME))
-        pwdb_dataframe = pd.DataFrame.from_records(pwdb_json_dataset)
-        pwdb_dataframe_columns = self.prepare_pwdb_data(pwdb_dataframe)
+        pwdb_dataset = Dataset.PWDB.fetch()
+        pwdb_dataframe_columns = self.prepare_pwdb_data(pwdb_dataset)
         pwdb_target_groups_refactor = self.target_group_refactoring(pwdb_dataframe_columns)
         pwdb_word2vec_model = self.train_pwdb_word2vec_language_model(pwdb_target_groups_refactor)
         pwdb_train_test_data = self.train_pwdb_data(pwdb_target_groups_refactor)
```

Comment:
we no longer train any language model on our data. 
this operation shall be discarded. We shall simply use a CHOSEN language model. How the choice is done, shall be parametrised: e.g. law2vec over jr word embedidngs, vs .. something else.